app/main.py

```diff
@@ -1,3 +1,42 @@
 class Distance:
-    # Write your code here
-    pass
+    def __init__(self, km: int | float) -> None:
+        self.km = km
+
+    def __str__(self) -> str:
+        return f"Distance: {self.km} kilometers."
+
+    def __repr__(self) -> str:
+        return f"Distance(km={self.km})"
+
+    def __add__(self, other: object) -> object:
```

Comment:
`other: object` and `-> object` 
Is not accepted as type annotations in this task. Please specify the correct type annotations